backends/smt2/smtbmc.py

```diff
@@ -969,6 +975,38 @@ def write_vcd_trace(steps_start, steps_stop, index):
 
         vcd.set_time(steps_stop)
 
+def find_state_loop(steps_start, steps_stop):
+    print_msg(f"Looking for loops")
+
+    path_list = list()
+
+    for netpath in sorted(smt.hiernets(topmod)):
+        hidden_net = False
+        for n in netpath:
+            if n.startswith("$"):
+                hidden_net = True
+        if not hidden_net:
```

Comment:
I picked the easy way, because it seems like handling a strange case that would take some time to write tests for, requiring more than my way of "mess around with examples/smtbmc and see if it does what I think it should". Should I open an issue?